docs/api.html

```diff
@@ -8015,6 +8023,46 @@
         "tags" : [ "/api/ldap" ]
       }
     },
+    "/api/logger/adjust" : {
+      "post" : {
+        "summary" : "POST /api/logger/adjust",
+        "description" : "Get the namespaces of preset `id`.",
+        "parameters" : [ ],
+        "requestBody" : {
+          "content" : {
+            "application/json" : {
+              "schema" : {
+                "type" : "object",
+                "properties" : {
+                  "duration" : {
+                    "type" : "integer"
+                  },
+                  "duration_unit" : {
+                    "$ref" : "#/components/schemas/metabase.api.logger~1time-unit"
+                  },
+                  "log_levels" : {
+                    "type" : "object",
+                    "additionalProperties" : {
+                      "$ref" : "#/components/schemas/metabase.api.logger~1log-level"
+                    }
+                  }
+                },
+                "required" : [ "duration", "duration_unit", "log_levels" ]
```

Comment:
> The `DELETE` endpoint mentioned in SEM-219 has not been implemented since it would have the same effect as the `POST` with an empty `log_levels` map.

I understand `duration` and `duration_unit` won't matter in case of empty `log_levels` map.
Can we make these params optional?

Reply:
Correct, there will be nothing to undo in that case. I can make them optional in that case. The generated documentation would get more misleading though, because you'll just see that those fields are optional in general, which in most cases they aren't. WDYT?

Reply:
I called the endpoint twice in a row with empty `log_levels` map and now it's always giving me an error:
`"Cannot invoke \"java.util.concurrent.Future.cancel(boolean)\" because \"undo_task\" is null"`

Reply:
Should work now, but there is a DELETE endpoint now too.